### ISSUE TITLE
About the setting in ablataion studies. 

### ISSUE THREAD

**Issue by Solacex** [chenfengxu714/SqueezeSegV3#19]
Hello, 

Thanks for your inspiring work~

In the Sec. 6.4 of the main paper, you mentioned that the ablation analyses are conducted with shrunk LiDAR images (64*512). Could you share more details with us? change the image_prop[hight][width] directly? 
Besides, it would be grateful if you could share us with the hyperparameters of training under this setting. (Batch size, learning rate, epochs, etc).

Thanks again for your insightful work.

**Comment by chenfengxu714** [comment]
Thank you. Yes, you can directly change the image_prop[hight][width]. As for the hyperparameters of training under this setting, I am not quite sure since it has been a long time. We didn't carefully tune these hyperparameters before, so I think you maybe just need to keep the same training recipe, it will reflect the similar conclusions.

**Comment by Solacex** [comment]
Thanks for your quick response~
